src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,24 @@
+@import "../utils/fonts";
+
+body {
+  font-family: "Poppins", sans-serif;
+  background-color: $gray-color;
+  color: $white-color;
+}
+
+h2 {
+  font-weight: 500;
+  font-size: 32px;
+  letter-spacing: -1.5px;
+  color: $white-color;
+  line-height: 100%;
+
+  text-align: center;
+
+  @include onTablet {
+    font-size: 48px;
+  }
+  @include onDesktop {
+    font-size: 64px;
+  }
+}
```

Comment:
Don't stylize tag selectors. Create a separate file called `typography` with the `.h2` or `.heading2` selector.